sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/CategorizedEntity.java

```diff
@@ -104,7 +84,17 @@ public String getSubcategory() {
     }
 
     /**
-     * Get the offset of entity text.
+     * Get the score property: If a well-known item is recognized, a decimal
+     * number denoting the confidence level between 0 and 1 will be returned.
+     *
+     * @return The score value.
+     */
+    public double getConfidenceScore() {
+        return this.confidenceScore;
+    }
+
+    /**
+     * Get the offset of entity text. The start position for the entity text in a document.
```

Comment:
I'm a little confused, since it appears that `offset` and `length` seemed to be already exposed for `CategorizedEntity` and `LinkedEntityMatch`, but not for `SentenceSentiment`. Could be reading this wrong, but I expected `offset` and `length` not to be exposed here before this PR

Reply:
The PRs for adding `CategorizedEntity` and `LinkedEntityMatch` has merged in preview PR. But since the Opinion Mining PR hasn't merged in that point and trying to have less code conflicting, I wasn't done the offset and length at the same time of `CategorizedEntity` and `LinkedEntityMatch`. You can see what changes in CHANGELOG.  It should be fine and Sorry for the confusion.

Reply:
No problem, thanks for the clarification!